### ISSUE TITLE
Allow user to choose audio edition

### ISSUE THREAD

**Issue by sqiouyilu** [sqiouyilu/stylemy.name#4]
The default version should be semantically formatted so that screen readers can do TTS, and a human-read audio version should also be available for accessibility, ideally with a speed toggle setting that persists between questions.

To keep file size of the actual generator small, audio should not be embedded in the page but linked to and played on interaction. People who want to have an offline version should be able to download a package that includes all the audio files.
